packages/gatsby-theme-newrelic/src/components/SEO.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import { graphql, useStaticQuery } from 'gatsby';
+import useLocale from '../hooks/useLocale';
+const path = require('path');
+
+const SEO = ({ title, location }) => {
```

Comment:
We should also add a `children` prop to allow for individual sites to add additional values. For example, the developer site adds a bunch of Swiftype related stuff here: https://github.com/newrelic/developer-website/blob/develop/src/components/Seo.js#L45-L76

Reply:
I was thinking along those lines as well! I like that idea, makes it easier to migrate over / if we ever want to abstract anything back into the theme